pkg/cmd/experimental/syncgroups/config/types.go

```diff
@@ -0,0 +1,148 @@
+package config
+
+import (
+	ldaputil "github.com/openshift/origin/pkg/auth/authenticator/password/ldappassword/util"
+	"github.com/openshift/origin/pkg/client"
+	"k8s.io/kubernetes/pkg/util"
+)
+
+// LDAPSyncCommonConfig contains all of the config that is common to all LDAP schemas
+type LDAPSyncCommonConfig struct {
```

Comment:
But this object *doesn't* have that information. Look at the schema-specific `LDAPSyncConfig` structs - there's a reason I put the other two schemas in this pull, so it was obvious where I was going. This is just all of the config that will be the same for every schema. The utility of this object is so that I don't repeat myself on all of those pieces for each schema config.